gradle.properties

```diff
@@ -30,7 +30,7 @@ spark13Version = 1.6.2
 spark20Version = 2.3.0
 spark22Version = 2.2.3
 spark24Version = 2.4.4
-spark30Version = 3.0.1
+spark30Version = 3.2.0
```

Comment:
This is a minor thing, but should we call this spark32Version, or spark3Version? Seems odd (and inconsistent with what we've done in spark 2.x) to have the spark 3.0 version be 3.2.